src/ctia/http/routes/common.clj

```diff
@@ -0,0 +1,42 @@
+(ns ctia.http.routes.common
+  (:require [clojure.string :as str]
+            [clj-http.headers :refer [canonicalize]]
+            [ring.util.http-status :refer [ok]]
+            [schema.core :as s]
+            [ring.swagger.schema :refer [describe]]))
+
+(s/defschema PagingParams
+  {(s/optional-key :sort_by) (describe (s/enum :id) "Sort results on a field")
+   (s/optional-key :sort_order) (describe (s/enum :asc :desc) "Sort direction")
+   (s/optional-key :offset) (describe Long "Pagination Offset")
+   (s/optional-key :limit) (describe Long "Pagination Limit")})
+
+(defn map->paging-header-value [m]
+  (str/join "&" (map (fn [[k v]]
+                       (str (name k) "=" v)) m)))
+
+(defn map->paging-headers
```

Comment:
Why as headers and not in the body itself?  I would rather have PaginatedJudgements and other appropriate wrapper objects.  This is not a blocker for the merge, IMO.